src/rqt_launch/node_controller.py

```diff
@@ -84,7 +84,7 @@ def stop(self):
     def check_process_status(self):
         if self._proxy.has_died():
             rospy.logerr("Process died: {}".format(
-                                                  self._proxy.get_proc_name()))
+                self._proxy.get_proc_name()))
```

Comment:
This line doesn't need to be broken

Reply:
The break is necessary to keep it under 79 characters as specified in PEP 8.